apps/financial-aid/web-osk/src/routes/application/TaxReturnForm/taxFormContent.tsx

```diff
@@ -0,0 +1,118 @@
+import { Text, LinkContext } from '@island.is/island-ui/core'
+
+const DirectTaxPaymentsInfo = () => {
+  return (
+    <>
+      <Text as="h2" variant="h3" marginBottom={2}>
+        Hvar finn ég staðfestingarskjal úr staðgreiðsluskrá?
+      </Text>
+      <Text marginBottom={[3, 3, 10]}>
+        Eftir að þú hefur innskráð þig á Þjónustuvef Skattsins ferð þú í Almennt
+        → Staðgreiðsluskrá RSK → Sækja PDF.
+      </Text>
+    </>
+  )
+}
+
+const TaxReturnInfo = () => {
+  return (
+    <>
+      <Text as="h2" variant="h3" marginBottom={2}>
+        Hvar finn ég staðfest afrit af mínu skattframtali?
+      </Text>
+
+      <LinkContext.Provider
+        value={{
+          linkRenderer: (href, children) => (
+            <a
+              style={{
+                color: '#0061ff',
+              }}
+              href={href}
+              rel="noopener noreferrer"
+              target="_blank"
+            >
+              {children}
+            </a>
+          ),
+        }}
+      >
+        <Text marginBottom={[3, 3, 5]}>
+          Á vef Skattsins finnur þú{' '}
+          <a href="https://www.skatturinn.is/einstaklingar/framtal-og-alagning/stadfest-afrit-framtals/">
```

Comment:
How will this link appear? Is it blue and underlined or does it just look like normal text? If it just looks like normal text then I was thinking of using the Link component to make it blue and underlined so it is more obvious it is a link.